codegen-winter/src/air/public_inputs.rs

```diff
@@ -0,0 +1,27 @@
+use super::Scope;
+
+/// Updates the provided scope with a public inputs.
+pub(super) fn add_public_inputs_struct(scope: &mut Scope, name: &str) {
+    // define the custom Air struct.
+    scope
+        .new_struct(name)
+        .vis("pub")
+        .field("program_hash", "Digest")
+        .field("stack_inputs", "Vec<Felt>")
+        .field("stack_outputs", "Vec<u64>")
+        .field("overflow_addrs", "Vec<u64>");
```

Comment:
Similar to the comment I left in the other spots - these need to be dynamically defined based on the provided AirIR

Reply:
Fixed.